api/package.json

```diff
@@ -80,7 +80,6 @@
 		"@directus/drive-azure": "9.0.0",
 		"@directus/drive-gcs": "9.0.0",
 		"@directus/drive-s3": "9.0.0",
-		"@directus/extensions-sdk": "9.0.0",
```

Comment:
The API still needs to depend on the extensions sdk so API extensions can import helpers like `defineHook` and `defineEndpoint`. Maybe we should make this a peer dependency instead?

Reply:
Ah, thanks for this! Yes, I think in this case it makes sense to add it as a peer dependency.